### ISSUE TITLE
Unable to open settings / key bindings

### ISSUE THREAD

**Issue by srdjan-m** [maegul/comment_marks#1]
![image](https://user-images.githubusercontent.com/2496599/150751218-a5324dd5-598a-4f83-be76-daa5d01e539b.png)
I am not able to change neither the settings nor key bindings.
OSX 12.1

**Comment by maegul** [comment]
Restarting sublime is necessary for the package to work. Annoying I know, hopefully I’ll work out a way around it one day. 

Let me know if it doesn’t help.

**Comment by maegul** [comment]
Actually, @srdjan-m ... there is a bug here with the menu items.  Sorry!

It should be fixed in the newest version (`0.1.10`).  Once `Package Control` picks up the new version you can run `Package Control > Upgrade Package`.

I don't know how long it takes Package Control to pickup updates.

In the meantime, here are some workarounds:

* **To change the settings**, the command palette command still works: `Command Palette`>`Comment Marks: Settings`
* **To change the keybindings**, all you have to do is add keybindings to your user keybindings file.
  * From the `Command Palette`: `Preferences: Key Bindings`
  * Then add the following keybinding, freely modifying the keys as you see fit:

*Note that this must go within the master array/list of commands that should already be defined*

```json
	{
		"command": "goto_comment",
		"keys": ["super+k","super+shift+r"],
	}
```

**Comment by maegul** [comment]
@srdjan-m, The fix appears to be working as far as I can tell so I'll close this now.  Thanks for picking up on the problem!
